### PR TITLE
Driver Benchmark tests

### PR DESCRIPTION
**Pull request by estolfo**

### CODE REVIEW

File: profile/benchmarking/multi_doc.rb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+          Benchmark.realtime do
+            collection.insert_many(docs)
+          end
+        end
+        client.database.drop
+        Benchmarking.median(results)
+      end
+
+      def client
+        @client ||= Mongo::Client.new(["localhost:27017"], database: 'perftest')
```
Comment (durran) [inline profile/benchmarking/multi_doc.rb]: Can add option `:monitoring => false` to speed up as well.

Comment (estolfo) [inline profile/benchmarking/multi_doc.rb]: ah, good point. Thanks

**Comment by durran** [comment]
Are we commiting the json data files as well?

**Comment by estolfo** [comment]
So far I wasn't committing the data files, as some are quite large and I think they're on s3. I can commit the smaller ones though.

**Pull request merged**